docs/website/docs/blog/2021-07-19-tflite-tosa.md

```diff
@@ -0,0 +1,38 @@
+## TFLite Support via TOSA
+
+IREE can now execute [TensorFlow Lite](https://www.tensorflow.org/lite)
+(TFLite) models through the use of
+[TOSA](https://developer.mlplatform.org/w/tosa/), an open standard of common
+tensor operations, and a part of [MLIR](https://mlir.llvm.org/) core. TOSA’s
+high-level representation of tensor operations provides a common front-end for
+ingesting models from different front-ends. In this case we ingest a TFLite
+flatbuffer, compile it to TOSA IR which IREE takes as an input format to compile
+to its various backends.
+
+![Compilation diagram](https://https://github.com/google/iree/tree/main/docs/website/docs/blog/2021-07-19-tflite-tosa-compilation-diagram.png?raw=true){ align=center }
+
+Executing TFLite on IREE provides an alternative execution environment to the
+TFLite runtime that can leverage IREE’s advantages, specifically multi-cpu and
+GPU architecture support using a lightweight execution environment. Having this
+compile-once-run-everywhere approach guarantees more consistent performance on a
```

Comment:
Ehhh.. "compile-once-run-everywhere" sounds nice, but that's not quite how compilation works :P 

Some of the benefits of ahead-of-time compilation: 
* you can specialize by compiling for a specific target (e.g. Android architecture)
* more opportunities to optimize: "All tensor operations are aggressively fused to avoid the need for transient memory, runtime memory planning (even when using dynamic shapes), or copies." (https://github.com/google/iree/tree/main/bindings/tflite#%EF%B8%8F-tflite-micro)
* only the code that is needed for each model is included in your final binary, without any loss of generality (no need for any of https://www.tensorflow.org/lite/guide/reduce_binary_size)
  * since the runtime and model files are both self-contained, you can load different models without needing to push updates to your app, all while still enjoying those binary size savings

Reply:
Overall we should focus on succinctness for this description - remember the discussion is not what benefits IREE has an execution environment but rather why support TFLite/TOSA as an ingestion formats. Beyond that, the benefits should be very succinct - smaller binary size, better performance, etc.

Reply:
Yep, I agree that a succinct description here would be good - just wanted to point out that "compile-once-run-everywhere" isn't accurate and offer some ideas for other points to make.